<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralCleaner</class>
 <widget class="QMainWindow" name="GeneralCleaner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General Cleaner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>General Cleaner</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="imageFileUserlistWidget"/>
       </item>
       <item>
        <widget class="QLabel" name="currentFilelabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DrAmtlabel_2">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>AddUser()</slot>
  <slot>DeleteUser()</slot>
  <slot>EditUser()</slot>
 </slots>
</ui>
